ci-operator/job/openshift/etcd/openshift-etcd-origin-4.0-etcd-v3.3.10-postsubmits.yaml

```diff
@@ -0,0 +1,35 @@
+postsubmits:
+  openshift/etcd:
+  - agent: kubernetes
+    branches:
+    - ^origin-4\.0-etcd-v3\.3\.10$
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/variant: rhel
+    name: branch-ci-openshift-etcd-origin-4.0-etcd-v3.3.10-rhel-images
```

Comment:
You'll need to delete all the job/* files and let ci-operator-prowgen recreate them

Reply:
ack